src/actions/layout.js

```diff
@@ -1,13 +1,21 @@
-import { CLOSE_SETTINGS, OPEN_SETTINGS } from '../types';
+import { TOGGLE_SETTINGS, TOGGLE_LOADER, TOGGLE_SIDE_MENU } from '../types';
 
-const openSettings = () => (dispatch) =>
+const toggleSettings = (showSettings) => (dispatch) =>
   dispatch({
-    type: OPEN_SETTINGS,
+    type: TOGGLE_SETTINGS,
+    payload: showSettings,
   });
 
-const closeSettings = () => (dispatch) =>
+const toggleLoader = (showLoader) => (dispatch) =>
```

Comment:
This is for the `ProgressScreen`, right? Maybe gets confused with the `Loader` component. Let's be consistent. I think renaming the component to `LoadingScreen` and then using `toggleLoadingScreen` and `showLoadingScreen` would be clearer.